### ISSUE TITLE
Cloudflare (failed with status code 403)

### ISSUE THREAD

**Issue by Mix-Anik** [Mix-Anik/chatgpt-lib#3]
chat.openai.com started using cloudflare, which breaks current logic and prevents library from making requests without passing cloudflare check returning 403.

**Comment by jcyuan** [comment]
will this fixed soon?

**Comment by Mix-Anik** [comment]
yes, i will try to fix it by the end of weekends, there already are some solutions out there, i just didn't quite had time to deal with that issue yet

**Comment by Mix-Anik** [comment]
Unfortunately i will not be able to make progress under current circumstances with openai's new anti-spam protections, but they are just too much for such small project to make without guarantee there wouldn't be further protective oppressions from openai's side.

I will archive this repo, but if you're still willing to build something around chatgpt's unofficial api in javascript, try out
https://github.com/transitive-bullshit/chatgpt-api
They seem to have implemented solutions, at least to some degree, for bypassing current protections, even tho some of them may require payment to be done (recaptchas), which at this point i would consider using openai's available api plan for davinci or some other model.
